pootle/apps/pootle_store/unit/altsrc.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from pootle.i18n.gettext import language_dir
+from pootle_store.templatetags.store_tags import pluralize_target
+
+from .proxy import UnitProxy
+
+
+class AltSrcUnitProxy(UnitProxy):
+    @property
+    def store_path(self):
+        """Returns just the path part omitting language and project codes."""
+        return self.unit["store__pootle_path"].split(u'/', 2)[-1]
```

Comment:
hmm... this is copy-paste from `store.path` which was used before. So I wonder how it worked.